GETTING_STARTED.md

```diff
@@ -1,109 +1,86 @@
 # Getting Started
 
-The SDK supports three ways for authentication:
-
-1. Using the email and password:
+## Auth - Connect to GDN
+The first step to start using GDN is establishing a connection to [gdn.paas.macrometa.io](https://gdn.paas.macrometa.io/). When this code executes, it initializes the server connection to your nearest region. You can access your Macrometa GDN account using several methods, such as [API keys](https://macrometa.com/docs/account-management/api-keys/), [User authentication](https://macrometa.com/docs/account-management/auth/user-auth), [Token-based authentication (JWT)](https://macrometa.com/docs/account-management/auth/jwts). For this example the method used is API key.
 
 ```python
-  # Authentication email and password
-  client = C8Client(protocol='https', host='gdn.macrometa.io', port=443,
-   email="user@example.com", password="XXXXX")
+from c8 import C8Client
+client = C8Client(protocol='https', host='gdn.paas.macrometa.io', port=443, apikey=<your api key>)
```

Comment:
I have a question here for everyone.. should we just show one way of connect to GDN? shouldn't we show the Token way and email password way too?

Reply:
We can have it, but I don't think that it will do much difference. The hyperlink is there for other methods